apps/angular/src/app/app.component.html

```diff
@@ -1 +1,6 @@
-<router-outlet></router-outlet>
+<div class="app">
+  <camp-header class="app__header"></camp-header>
+  <main class="app__content">
+    <router-outlet></router-outlet>
+  </main>
+</div>
```

Comment:
I think, it is a bit wrong to create `div` only to follow right BEM methodology. And I see these classes are not used in CSS file.
```suggestion
<camp-header></camp-header>
<main>
  <router-outlet></router-outlet>
</main>
```